### DISCUSSION TITLE
Recommendations for Multi-shell DWI scans for anisotropic simulations

### DISCUSSION THREAD

**Discussion by gruberanthony** [simnibs/simnibs#288] [7.Creating head models]
I am looking for recommendations regarding the use of DTI in SimNibs. My current understanding is that SimNibs 4.0.0 does not support running multi-shell scans through DWI2Cond unless they have been preprocessed beforehand. The issue I have been running into is that most of the softwares I use to run the preprocessing do not create a native nifti file that FSL requires. For example, when I use MRtrix3, I then have to run tckmap to create the nifti file and then run that file through dtifit. However, based on FSL's recommendations, dtifit creates tensors that are not a good fit for multi-shell data. So in this instance, should I extract volumes with a single b-value using the select_dwi_vols command to ensure data quality or would it be fine to simply run the multi-shell scan through dtifit to then run the anisotropic analysis. Any recommendations are appreciated.

**Comment by axelthi** [comment]
Hi,
sorry for the very late answer. Your question is likely not only related to specifics of how dtifit handles multishell data, but the general question whether diffusion tensors estimated from multi-shell data correlate with extracellular ion diffusion (that determines tissue conductivity) worse or better than single-shell data. I don't think that this is well understood so far. Personally, I would prefer to extract single-shell data at a lower b-value (800 or 1000) to stay in the range where Gaussian diffusion patterns are expected (Jones & Basser, MRM 2004) and for which the relation between water and ion diffusion (Tuch et al PNAS 2001) was originally suggested.
